src/containers/App.js

```diff
@@ -193,7 +196,14 @@ class App extends React.Component {
           <div className={styles.entriesPanel}>
             { isFetching && <TopBarProgress /> }
             <div>
-              {children}
+              <Switch>
+                <Route exact path='/' component={DashboardPage} />
+                <Route exact path="/collections/:name" component={CollectionPage} />
+                <Route path="/collections/:name/entries/new" render={(props) => (<EntryPage {...props} newRecord={true}/>)} />
```

Comment:
One thing I'm not quite sure about is whether this is the best way to pass `newRecord` through here.

Reply:
I think the most "proper" way to do this would be to move the actual logic for `newRecord` to EntryPage itself. @erquhart @Benaiah do you mind that?

Reply:
You could clean it up a little: `render={props => <EntryPage {...props} newRecord />}`

I think the boolean prop will work that way, right?